tests/sles4sap/publiccloud/hana_sr_takeover.pm

```diff
@@ -50,6 +50,12 @@ sub run {
     # Calculate SBD delay sleep time
     $sbd_delay = $self->sbd_delay_formula if $takeover_action eq 'crash';
 
+    # SBD delay related setup for 'stop' to fix sporadic 'takeover failed to complete' issue on EC2
+    if ($takeover_action eq 'stop' and check_var('PUBLIC_CLOUD_PROVIDER', 'EC2')) {
+        $self->setup_sbd_delay();
+        $sbd_delay = $self->sbd_delay_formula();
```

Comment:
`setup_sbd_delay` returns delay that is set up so you can do only 
``` $sbd_delay =  $self->setup_sbd_delay(); ```
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/8af56eeec5ec82a8833f867bcf45ea284525feea/lib/sles4sap_publiccloud.pm#L502C19-L502C19